app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/rootnav/RootNavViewModel.kt

```diff
@@ -63,8 +63,7 @@ class RootNavViewModel @Inject constructor(
         val specialCircumstance = action.specialCircumstance
         val updatedRootNavState = when {
             userState?.activeAccount?.trustedDevice?.isDeviceTrusted == false &&
-                !userState.activeAccount.isVaultUnlocked &&
-                !userState.activeAccount.hasManualUnlockMechanism -> RootNavState.TrustedDevice
+                !userState.activeAccount.isVaultUnlocked -> RootNavState.TrustedDevice
```

Comment:
This was another issue I ran into where the user would go to the `VaultUnlockScreen`, when they should actually be prompted with the Trusted Device Screen.